### PR TITLE
Update logging revert output cleanup image

### PR DESCRIPTION
**Pull request by rewiko**
Let's reintroduce the logging.

### CODE REVIEW

File: script/cron-cleanup-docker-registry-image.sh [hunk 1]
```diff
@@ -17,14 +17,15 @@ REGISTRY_DATA_DIR=${registryVolume} /root/script/clean_old_versions.py \
                                     --image "${patternImage}" \
                                     -l ${nbTagToKeep} \
                                     --order "${orderBeforeDeletion}" \
-                                    --registry-url ${registryUrl}
+                                    --registry-url ${registryUrl} > /var/log/clean-images-registry.log
```
Comment (sebbonnet) [inline script/cron-cleanup-docker-registry-image.sh]: should be `>>` too here, so we keep on adding for each cleanup run

Comment (rewiko) [reply script/cron-cleanup-docker-registry-image.sh]: the pod is deleted anyway

Comment (sebbonnet) [reply script/cron-cleanup-docker-registry-image.sh]: safeguards against future modifications to the script which could add more logging :-P

**Comment by rewiko** [comment]
Sir, I've updated this PR, to match your requirements.

**Review by sebbonnet — APPROVED** [review]

**Pull request merged**